charts/velocity-operator/templates/manager-rbac.yaml

```diff
@@ -9,7 +9,7 @@ rules:
   - "" # core api group
   - apps
   - extensions
-  - networking.k8s.io/v1
+  - networking.k8s.io
```

Comment:
Does this include `v1` also?

Reply:
`networking.k8s.io` is the api group name rather than version

Reply:
For example this would work? https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/service/networking/minimal-ingress.yaml

Reply:
```apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: frontend-ingress
spec:
  rules:
    - host: frontend-{velocity.v1.domainSuffix}
      http:
        paths:
          - backend:
              service:
                name: frontend
                port:
                  number: 8080
            path: /
            pathType: Prefix
  tls:
    - hosts:
        - frontend-{velocity.v1.domainSuffix}
      secretName: wildcard-cert```

This worked